packages/shared/pkg/logs/logger.go

```diff
@@ -179,6 +179,26 @@ func (l *SandboxLogger) MemoryUsage(memoryMB float64) {
 	}
 }
 
+func (l *SandboxLogger) CPUPct(cpuPct float64) {
+	l.exporter.logger.Info().
+		Str("instanceID", l.instanceID).
+		Str("envID", l.envID).
+		Str("teamID", l.teamID).
+		Float64("cpuPct", cpuPct).
+		Int32("cpuCount", l.cpuMax).
+		Msg("CPU usage")
+}
+
+func (l *SandboxLogger) MemMB(memMB uint64) {
+	l.exporter.logger.Info().
+		Str("instanceID", l.instanceID).
+		Str("envID", l.envID).
+		Str("teamID", l.teamID).
+		Uint64("memMBUsed", memMB).
+		Int32("memoryMBTotal", l.memoryMBMax).
```

Comment:
Is it converted to the same type? It would probably make sense to have it as same type (only exception would be using float)